README.md

```diff
@@ -69,3 +69,17 @@ tsconfig.base.json    // common typescript configuration
 tslint.json           // monorepo wide linting configuration
 yarn.lock             // the only lock file in the repo. all packages combined
 ```
+
+## Monorepo methodology
+
+Development in a monorepo environment should take place in the root of the project. However, users can consume specific packages (as opposed to consuming the entire monorepo) if they'd like. This methodology defines a structure for dependencies:
+- dev-dependencies should be placed in the root `package.json`
+- dependencies and peer-dependencies should be placed the `package.json` of the relevant package
+
+Additionally, some packages depend on sibling packages within the monorepo. For example, `app` depends on `components`. This relationship should be described in the `package.json` of `app` like so:
+
+```json
+  "dependencies": {
+    "components": "*"
```

Comment:
the * is only if you didn't publish yet.
technically, all packages in the monorepo got access to all packages.

Reply:
So add that as a note?

Reply:
It's just not clarified enough... the * is temporary until ypu publish. Then, it becomes an actual compatible version (when publishing). this probably needs to be explained.

Reply:
@Zombiefruit removed the * from master.

Now they specify versions even if not published. It just makes sense.

Reply:
Sweet. I'll update ui-autotools to do the same